src/components/Person/Person.jsx

```diff
@@ -1 +1,40 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
```

Comment:
```suggestion
  const { 
    name, 
    age, 
    sex, 
    isMarried, 
    partnerName 
  } = person;
```

for better readability